src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -2,18 +2,106 @@
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] table;
+    private int size;
+    private Node<K, V> nullKeyNode;
+
+    public MyHashMap() {
+        this.table = new Node[DEFAULT_CAPACITY];
+    }
+
+    private static class Node<K, V> {
+        private final K key;
+        private V value;
+        private Node<K, V> next;
+
+        Node(K key, V value) {
+            this.key = key;
+            this.value = value;
+        }
+    }
+
+    int getBucketIndex(K key) {
```

Comment:
```suggestion
    private int getBucketIndex(K key) {
```
Replace after public methods